src/ethereum_test_exceptions/geth_exceptions.py

```diff
@@ -0,0 +1,140 @@
+"""
+Geth exceptions ENUM -> str mapper
+"""
+
+from .exception_mapper import ExceptionMapper, ExceptionMessage
+from .exceptions import EOFException, TransactionException
+
+
+class GethExceptionMapper(ExceptionMapper):
```

Comment:
These are very directly related to the Geth T8N implementation, it might be a good idea to simply move it into `src/evm_transition_tool/geth.py`.

Same for the `evmone_exceptions.py`.

Reply:
Yes. This can also be a base exception mapper.
Then geth t8n defines geth exception mapper for t8n.
And geth consume defines exception mapper for consume driver.